diff --git a/dist/cjs/client.js b/dist/cjs/client.js
index 191eb1ac850f4cba745f87505374a2848f477d0d..197f2bb699cbe646340ff7461fb104402ed59f65 100644
--- a/dist/cjs/client.js
+++ b/dist/cjs/client.js
@@ -2293,13 +2293,6 @@ class CopilotClient {
       }
       this.cliProcess.stderr?.on("data", (data) => {
         this.stderrBuffer += data.toString();
-        const lines = data.toString().split("\n");
-        for (const line of lines) {
-          if (line.trim()) {
-            process.stderr.write(`[CLI subprocess] ${line}
-`);
-          }
-        }
       });
       this.cliProcess.on("error", (error) => {
         if (!resolved) {
diff --git a/dist/client.js b/dist/client.js
index c9eaedbf00d592f2f3681ac59cb48dca59aaa130..2ccd85ae637193c29f77fd1500c71155afd487fb 100644
--- a/dist/client.js
+++ b/dist/client.js
@@ -2273,13 +2273,6 @@ class CopilotClient {
       }
       this.cliProcess.stderr?.on("data", (data) => {
         this.stderrBuffer += data.toString();
-        const lines = data.toString().split("\n");
-        for (const line of lines) {
-          if (line.trim()) {
-            process.stderr.write(`[CLI subprocess] ${line}
-`);
-          }
-        }
       });
       this.cliProcess.on("error", (error) => {
         if (!resolved) {
